contrib/fake-openshift-ansible/fake-openshift-ansible

```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/bash
+
+# Ensure that the playbook file exists.
+[ -z $PLAYBOOK_FILE ] && { echo "PLAYBOOK_FILE environment variable not set."; exit 1; }
+[ ! -f "${PLAYBOOK_FILE}" ] && { echo "Playbook file ${PLAYBOOK_FILE} not found."; exit 1; }
+
+# Ensure that the inventory file exists.
+[ -z $INVENTORY_FILE ] && { echo "INVENTORY_FILE environment variable not set."; exit 1; }
+[ ! -f "${INVENTORY_FILE}" ] && { echo "Inventory file ${INVENTORY_FILE} not found."; exit 1; }
+
+# Ensure that the AWS access key ID exists.
+[ -z $AWS_ACCESS_KEY_ID ] && { echo "AWS_ACCESS_KEY_ID environment variable not set."; exit 1; }
+
+# Ensure that the AWS secret access key exists.
+[ -z $AWS_SECRET_ACCESS_KEY ] && { echo "AWS_SECRET_ACCESS_KEY environment variable not set."; exit 1; }
```

Comment:
AWS vars would only be required for the infra and scale group provisioning, when we get to control plane install and such they wouldn't necessarily be set, but I guess we could just for consistency.

Reply:
I would assume that we would still pass them into the ansible playbook pod anyway. Otherwise the clust op has to have more knowledge about exactly how the playbooks are implemented.